src/Ipac_Table_Writer/Ipac_Table_Writer.cxx

```diff
@@ -274,6 +274,7 @@ void generate_and_write_default_comments(
             auto unit = props.attributes.find("unit");
             if (unit != props.attributes.end() && !unit->second.empty()) {
                 col_comment.append(" (").append(unit->second).append(")");
+                boost::replace_all(col_comment, "\n", " ");
```

Comment:
Maybe we should replace both `\r` and `\n`, in case some IPAC table parsers recognize both as line terminators?